charts/handbrake/2.0.0/questions.yaml

```
@@ -20,14 +20,13 @@ groups:
 portals:
   web_portal:
     protocols:
-      - "http"  # if SECURE_CONNECTION is set, this should be https, or not.. container will redirect all http to https anyway
+      - "$kubernetes-resource_configmap_main-portal_protocol"
     host:
-      - "$node_ip"
+      - "$kubernetes-resource_configmap_main-portal_host"
     ports:
-      - "$variable-service.port.nodePort"
+      - "$kubernetes-resource_configmap_main-portal_port"
 # UI
 questions:
-
   # Update Policy
```

```suggestion
  # Update Policy
  - variable: portal
    group: "Container Image"
    label: "Configure Portal Button"
    schema:
      type: dict
      hidden: true
      attrs:
        - variable: enabled
          label: "Enable"
          description: "enable the portal button"
          schema:
            hidden: true
            editable: false
            type: boolean
            default: true
        - variable: nodePortProtocol
          label: "Protocol when using NodePort"
          description: "Enter the protocol to use when using nodeport"
          schema:
            hidden: true
            editable: false
            type: string
            default: "http"
```

http because handbrake will redirect to https if secure_conection is set.

---

Thanks thats some solid reasoning behind it :)

---

That leave Collabora :)
I just checked the path for you, not sure if the http is done correctly atm.